datastore/emulator.go

```diff
@@ -53,23 +53,35 @@ func Emulator(ctx context.Context) <-chan struct{} {
 	// Set log formatter to process newlines since emulator gives a lot of output
 	l.Logger.SetFormatter(&log.TextFormatter{DisableQuote: true})
 
-	// Set up the command
+	// Set up
```

Comment:
Go style says that all comments should be full sentences, starting with a capital and ending with a full stop.
If that's too much comment for a single line of code, then there shouldn't be a comment for it.

Reply:
Agreed, this is a bad leftover. Fixed.